.github/workflows/run-test.yaml

```diff
@@ -12,13 +12,10 @@ jobs:
     strategy:
       matrix:
         python-version:
-          - "3.5"
-          - "3.6"
-          - "3.7"
           - "3.8"
           - "3.9"
           - "3.10"
-          - "pypy-3.7"
+          - "pypy-3.8"
```

Comment:
may as well add 3.11 and 3.12

Reply:
Happy to.

what about the travis file?

Reply:
I think trafis is dead (at least I see no sings of it running) so we could just remove that file. cc @vytas7